### ISSUE TITLE
Deploy does not always throw on failure

### ISSUE THREAD

**Issue by marcust** [alexec/docker-maven-plugin#78]
I have a failing deploy that looks like this on the command line:

```
[INFO] {"status":"Pushing","progressDetail":{},"id":"340d662c0db5"}{"errorDetail":{"message":"HTTP code 500 while uploading metadata: \"invalid character '\u003c' looking for beginning of value\""},"error":"HTTP code 500 while uploading metadata: \"invalid character '\u003c' looking for beginning of value\""}
```

I'm not sure if that is actually one line, but it does not throw and thus the build does not fail. 

I think that is due to the fact that throwExceptionIfThereIsAnError checks for beginning of line 
```
 if (l.startsWith("{\"errorDetail") && !l.equals("{\"errorDetail\":{}}")) {
```

which doesn't seem to be correct for commands with progress?

**Comment by marcust** [comment]
I guess that was fixed in 3ddc95a

**Comment by alexec** [comment]
I hope so, have you tested?
